### DISCUSSION TITLE
How can I get the previous 'new theme' version?

### DISCUSSION THREAD

**Discussion by Robot-44** [SpotX-Official/SpotX#396] [Question]
Recently I updated Spotx and I don't really like how the new left bar looks like the old one was better imo.
Here is a Preview for the previous 'New Theme' that I was having
![image](https://user-images.githubusercontent.com/91937543/235306565-7915910f-01b7-4b75-a5ad-d0be8690f4ad.png)

**Comment by amd64fox** [comment]
two options for solving the problem:

1) without reinstallation

![Анимация4](https://user-images.githubusercontent.com/62529699/235309648-7f4f1d49-3404-4668-8808-1353d61b4a28.gif)


2) You need to perform a clean installation of [version 1.99](https://docs.google.com/spreadsheets/d/1wztO1L4zvNykBRw7X4jxP8pvo11oQjT0O5DvZ_-S4Ok/edit#gid=0) and force install SpotX over this version or lower, also do not forget to block updates

**Reply by Robot-44** [reply]
Thank you so much! Your help is appreciated
it worked! I downloaded the old version and blocked the auto updates.
BTW, I already knew about the compact view option in the new version but I can't really adjust the size of the left side bar if I enable it. Also, there is no dedicated button for the lyrics same as the older version. 
where can send my suggestions?

**Reply by amd64fox** [reply]
> Also, there is no dedicated button for the lyrics same as the older version.

there was a similar question [here](https://github.com/amd64fox/SpotX/issues/395) recently

**Reply by Robot-44** [reply]
ah thank you! I redownload the newer version and used the command
However in compact layout if I try to reduce the size of the left sidebar this happens. 
is there a way that I can fix it or it's just like this?

https://user-images.githubusercontent.com/91937543/235316015-29dc107a-7cb6-479f-a420-8af0150509ea.mp4

**Reply by amd64fox** [reply]
If you're talking about collapsing the compact mode, it's a feature of the new left sidebar and cannot be disabled. The only way to do it is to revert back to the old version of Spotify.
